scripts/set_requester_pays.sh

```diff
@@ -32,13 +28,15 @@ echo "Gatorcounting for 10 seconds while iam change goes into effect"
 echo ""
 echo "NOTE: if you get an error message saying:"
```

Comment:
this isn't part of your PR, but after using this a bunch, I find the following long error message example confusing - could we remove the next line and change this one to be just 
`echo "NOTE: if you get an error message saying AccessDeniedExeption: 403"`